spacepy/pybats/__init__.py

```diff
@@ -18,7 +18,9 @@
 from functools import wraps
 
 from spacepy.datamodel import dmarray, SpaceData
+import datetime as dt
 import numpy as np
+import warnings
```

Comment:
Notes on [PEP8](https://peps.python.org/pep-0008/#imports) imports...

- Separate standard library / other libraries / internal
- I like having them alphabetized within each section, easier to tell what's going on